core/src/test/java/org/hibernate/ogm/backendtck/associations/manytoone/ManyToOneTest.java

```diff
@@ -42,8 +42,8 @@ public void testUnidirectionalManyToOne() throws Exception {
 		session.persist( emmanuel );
 		session.persist( jerome );
 		session.flush();
-		assertThat( getNumberOfEntities( sessions ) ).isEqualTo( 3 );
-		assertThat( getNumberOfAssociations( sessions ) ).isEqualTo( expectedAssociations() );
+		assertThat( getNumberOfEntities( session ) ).isEqualTo( 3 );
+		assertThat( getNumberOfAssociations( session ) ).isEqualTo( expectedAssociations() );
 		transaction.commit();
 		assertThat( getNumberOfEntities( sessions ) ).isEqualTo( 3 );
```

Comment:
Is there a reason why you didn't change all the calls and get rid of the SessionFactory based versions?